chapters/hpmor-chapter-001.tex

```diff
@@ -1,181 +1,296 @@
+Apenaŭ.
 
-Harry’s mouth twisted bitterly. He was treated well, probably better than most genetic fathers treated their own children. Harry had been sent to the best primary schools—and when that didn’t work out, he was provided with tutors from the endless pool of starving students. Always Harry had been encouraged to study whatever caught his attention, bought all the books that caught his fancy, sponsored in whatever maths or science competitions he entered. He was given anything reasonable that he wanted, except, maybe, the slightest shred of respect. A Doctor teaching biochemistry at Oxford could hardly be expected to listen to the advice of a little boy. You would listen to Show Interest, of course; that’s what a Good Parent would do, and so, if you conceived of yourself as a Good Parent, you would do it. But take a ten-year-old \emph{seriously}? Hardly.
+Kelkfoje Hari volis krii al sia patro.
 
-Sometimes Harry wanted to scream at his father.
+“Panjo,” Hari diris.
+“Se vi volas venki ĉi tiun argumenton kontraŭ Paĉjo, rigardu la duan ĉapitron de la unua libro de \emph{La Prelegoj de Fajnman pri Fiziko}.
+Estas citaĵo tie pri kiel filozofoj diras multon pri tio, kion scienco absolute bezonas, kaj tio estas tute erara, ĉar la sola regulo en scienco estas ke la fina arbitro estas observado—ke vi simple devu rigardi la mondon kaj raportu tion, kion vi vidas.
+Nu…de la supro de mia kapo, mi ne povas pensi kie oni povas trovi ion pri kiel estas idealo de scienco solvi aferojn per eksperimentoj anstataŭ per argumentoj—”
 
-“Mum,” Harry said. “If you want to win this argument with Dad, look in chapter two of the first book of the Feynman Lectures on Physics. There’s a quote there about how philosophers say a great deal about what science absolutely requires, and it is all wrong, because the only rule in science is that the final arbiter is observation—that you just have to look at the world and report what you see. Um…off the top of my head I can’t think of where to find something about how it’s an ideal of science to settle things by experiment instead of arguments—”
+Lia patrino rigardis malsupren al li kaj ridetis.
+“Dankon, Hari.
+Sed—” ŝia kapo leviĝis por fiksrigardi sian edzon.
+“Mi ne volas venki argumenton kontraŭ via patro.
+Mi volas ke, ke mia edzo aŭskultu sian edzinon kiu amas lin, kaj fidi ŝin nur ĉi-foje—”
 
-His mother looked down at him and smiled. “Thank you, Harry. But—” her head rose back up to stare at her husband. “I don’t want to win an argument with your father. I want my husband to, to listen to his wife who loves him, and trust her just this once—”
+Hari mallonge fermis la okulojn.
+\emph{Senespera.}
+Ambaŭ de liaj gepatroj estis simple senespera.
 
-Harry closed his eyes briefly. \emph{Hopeless.} Both of his parents were just hopeless.
+Nun liaj gepatroj estis enirantaj en unu el \emph{tiaj} argumentoj denove, patrino provis fari ke patro sentu sin kulpa, kaj patro provis fari ke patrino sentu sin stulta.
```

Comment:
Ĉu eble "enirantaj" ne estas la plej bona traduko de "getting in to"? Mi pensas ke tiu estas tro laŭvorte anstataŭ laŭsence.

Reply:
Vidu fe51953